LibraBFT/Base/KVMap.agda

```diff
@@ -51,6 +51,8 @@ module LibraBFT.Base.KVMap  where
    ∈KV-empty-⊥    : k ∈KV (empty {Val = Val}) → ⊥
 
    lookup         : Key → KVMap Key Val → Maybe Val
+   insert         : (k : Key)(v : Val)(kvm : KVMap Key Val)
```

Comment:
`kvm-insert-Haskell` appears to be redundant now. Also, just wanted to note than in the future we will need to change the postulated proofs later in the file to relate `insert` to `kvm-insert` and `kvm-update` (or define those in terms of `insert`). But that isn't needed for this PR.

Reply:
I added a "Contract on the radar" card.